bindings/python/test/geometry/planetodetic/object/test_polygon.py

```
@@ -0,0 +1,220 @@
+        polygon: Polygon = Polygon.simple(ellipsoid, outer_ring)
+        assert polygon.get_vertex_count() == len(outer_ring)
+
+    def test_get_vertices_success(
+        self,
+        ellipsoid: Ellipsoid,
+        outer_ring: list[Point],
+    ):
+        polygon: Polygon = Polygon.simple(ellipsoid, outer_ring)
+
+        assert len(polygon.get_vertices()) == len(outer_ring)
+        for input_vertex, output_vertex in zip(polygon.get_vertices(), outer_ring):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)